stub/stub.go

```diff
@@ -539,12 +563,15 @@ func (s *Server) readNetPackets() {
 	if s.Trace != nil {
 		s.Trace("Starting readNetPackets")
 	}
+	if *serverprofile != "" {
+		s.beginSrvProfile()
```

Comment:
After this line
defer func() {
s.endSrvProfile()
}

Leave the returns as is below i.e. don't change them to break